lib/kube/proxy/resource_rbac_test.go

```diff
@@ -128,6 +129,31 @@ func TestListPodRBAC(t *testing.T) {
 		},
 	)
 
+	userWithTraits, _ := testCtx.CreateUserWithTraitsAndRole(
```

Comment:
nit: assert `NoError` here?

Reply:
`CreateUserWithTraitsAndRole` returns the user and the created roles. it doesn't return error